### PR TITLE
now with consonant stacking as well

### PR DESCRIPTION
**Pull request by erroronline1**
i am still not satisfied with the style of the drawing code though. this should be possible in a tidier way... but i am too tired for today.

### CODE REVIEW

**Comment by erroronline1** [comment]
had a bit more time...

**Comment by erroronline1** [comment]
does reopening this pull request the last commits in between?

**Comment by ModisR** [comment]
I think pull requests are for a whole branch, as opposed to a specific set of commits, so if you add more commits to a branch before its PR is accepted, those new commits become part of the PR.

File: js/shermans.js [hunk 1]
```diff
@@ -8,13 +8,13 @@ function shermansBase(char) {
...
 		"j": ["j", "ph", "k", "l", "c", "n", "p", "m"],
 		"t": ["t", "wh", "sh", "r", "v", "w", "s"],
 		"th": ["th", "gh", "y", "z", "q", "qu", "x", "ng"]
 	};
-	let rtrn="";
-	Object.keys(scgtable).forEach(function (row){
+	let rtrn = "";
+	Object.keys(scgtable).forEach(function (row) {
 		if (scgtable[row].indexOf(char) > -1) rtrn = row;
 	});
```
Comment (ModisR) [inline js/shermans.js]: I remember you expressing uncertainly about arrow functions at one point. The syntax is just replace `function (row)` with `row =>`.

Comment (erroronline1) [reply js/shermans.js]: thanks! i'll change that immidiately. will show up next time.

**Pull request merged**